components/AppLayout/GlobalSearch.js

```diff
@@ -123,14 +138,14 @@ function GlobalSearch({ onIconClick }) {
             <div className={classes.resultEntry} onClick={navigate('messages')}>
               <div className={classes.iconWithText}>
```

Comment:
Removing `iconWithText` div and just use `resultEntry` as flex container should yield same result with simpler structure.

![image](https://user-images.githubusercontent.com/108608/85193402-c979eb00-b2fa-11ea-95b5-06fddd8c09b6.png)

Also, we may need to add `flex-shrink: 0` to `.right`.